examples/complete/versions.tf

```diff
@@ -0,0 +1,3 @@
+terraform {
```

Comment:
pin `aws` provider?

Reply:
I felt it too redundant, since we're testing `../../`